src/components/ProgressBar/ProgressBar.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div :class="['progress-bar', ...classes]">
+    <div
+      class="progress-bar__inner"
+      :style="{ width: `${percentComplete}%` }"
+    ></div>
+  </div>
+</template>
+
+<script>
+import ProgressBar from '../../constants/ProgressBar';
+
+const { Sizes, Variants } = ProgressBar;
```

Comment:
nit: this can be the following if you export them each as defaults. 

```suggestion
import { Sizes, Variants } from '../../constants/ProgressBar';
```

Reply:
yeah i was following the pattern Jon set on the Button constants here